Framework/Model/MASShortcutValidator.m

```diff
@@ -52,6 +52,13 @@ - (BOOL) isShortcutValid: (MASShortcut*) shortcut
 
 - (BOOL) isShortcut: (MASShortcut*) shortcut alreadyTakenInMenu: (NSMenu*) menu explanation: (NSString**) explanation
 {
+    // Allow menu items in the 'Services' menu to pass the validation.
+    // Menu items in the 'Services' menu don't appear with their key equivants.
+    // This prevents the user from being confused when they can't find the conflicting menu item.
+    if (menu == [NSApp servicesMenu]) {
```

Comment:
Thanks for the PR. I don’t think we should unconditionally ignore Services.

Reply:
This looks like a compromised solution, but it's the only way that I currently could found to solve this specific problem. I'll add some more information here.

Reply:
I mean the shortcut recorder should have an optional flag that would permit you to ignore Services.

Reply:
I'see if I can add a flag to control this.